src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -4479,23 +4479,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         either have the .exe/.dll extension or their publish status has been overriden in VS so they will show up in the PublishFiles collection.
         The PublishProtocol property is available only in .NET>=5 so we will used that to exclude .NET FX 4.X case.
       -->
-      <_ClickOnceTransitiveContentItemsTemp Include="@(_TransitiveItemsToCopyToOutputDirectory->'%(TargetPath)')" Condition="'$(PublishProtocol)' == 'ClickOnce'" >
+      <_ClickOnceTransitiveContentItemsTemp Include="@(_TransitiveItemsToCopyToOutputDirectory->'%(TargetPath)')" Condition="'$(PublishProtocol)' == 'ClickOnce' AND '%(_TransitiveItemsToCopyToOutputDirectory.CopyToPublishDirectory)' != 'Never'" >
```

This is fine, but consider whether using the new WithoutMetadataValue item function would be clearer
```suggestion
      <_ClickOnceTransitiveContentItemsTemp Include="@(_TransitiveItemsToCopyToOutputDirectory->WithoutMetadataValue('CopyToPublishDirectory', 'Never')->'%(TargetPath)')" Condition="'$(PublishProtocol)' == 'ClickOnce'" >
```

---

done.